src/types/RelayPricer.ts

```diff
@@ -0,0 +1,10 @@
+export interface SourceApi {
+    query(input: string, ouput: string): Promise<number>;
+}
+
+export type CoinBaseResponse = {
+    data: {
+        currency: string;
+        rates: Record<string, string>;
+    };
+};
```

Comment:
This should be inside the coinbase-specific file

Reply:
What would be your recommendation for the naming, I have some doubts about it. Since the class is called RelayPricer and I named the type file as RelayPricer also, this could be confusing? I also have seen that some of the Class does not have a specific file and they declare the types inside the class file but I dont like this approach.

Reply:
updated